src/ig/IGExporter.ts

```diff
@@ -330,68 +305,41 @@ export class IGExporter {
           }
         );
 
-        if (!this.shouldCopyFiles) {
-          if (existsSync(inputIndexXMLPagesPath) || existsSync(inputIndexXMLPageContentPath)) {
-            generation = 'html';
-          }
-          // Add user-provided index file to IG definition
-          this.ig.definition.page.page.push({
-            nameUrl: 'index.html',
-            title: 'Home',
-            generation
-          });
-          return;
+        if (
+          (!existsSync(inputIndexMarkdownPagesPath) && existsSync(inputIndexXMLPagesPath)) ||
+          (!existsSync(inputIndexMarkdownPageContentPath) &&
+            existsSync(inputIndexXMLPageContentPath))
+        ) {
```

Comment:
This is very nitpicky, but if you had `pages/index.md`, and `pagecontent/index.xml`, wouldn't this end up preferring `pagecontent/index.xml`? Which seems slightly wrong. I think we might want to prefer both `md` files first, and only then go to `xml`, so:
```javascript
if (
!existsSync(inputIndexMarkdownPagesPath) &&
!existsSync(inputIndexMarkdownPageContentPath) &&
(existsSync(inputIndexXMLPagesPath) || existsSync(inputIndexXMLPageContentPath))
```

Reply:
Yeah I think this is a good idea. We can just always prefer either `.md` file over either `.xml` file. I think I was still caught up on the pattern that we had established previously, but I think this makes more sense. Updated in e02ff58